frontend/src/pages/agreements/approve/ApproveAgreement.jsx

```diff
@@ -132,6 +134,12 @@ const ApproveAgreement = () => {
                 title={title}
                 subTitle={agreement.name}
             />
+
+            <DebugCode
```

Comment:
did we mean to leave in Debug Code component?

Reply:
I did not. A bunch of changes came into the ApproveAgreement.jsx when I merged main in. I haven't touched any front end files myself, aside to fix the things that were brought in by the merge. Really unclear why some of that work was able to be merged into main while failing the linter! I'll remove this.

Reply:
Removed